### DISCUSSION TITLE
我认为应该添加无限生成，而不仅仅是最高32张数量。请求作者添加！

### DISCUSSION THREAD

**Discussion by saosoo** [lllyasviel/Fooocus#295] [Ideas]
目前最高的数量为32张，但是当我睡觉开始生成之后，等第二天早上醒来查看的时候它仅仅只有32张作品，我觉得它浪费了一晚上的时间。如果它能像SD那样无限生成，我可以很放心的去睡觉同时它在不停的工作。当我第二天醒来的时候我有足够多的作品可以去挑选最完美的作品。
请求作者加上这个功能！

**Comment by z0brk** [comment]
just change `maximum` and `value`  to a big int
 https://github.com/lllyasviel/Fooocus/blob/7538b4d17b476ba59f64fa7a1e750277900a0ba9/webui.py#L61

example:
```python
 image_number = gr.Slider(label='Image Number', minimum=1, maximum=999999999, step=1, value=999999999) 
```

**Reply by saosoo** [reply]
非常感谢大神的回复。这对于我来说非常有用。非常感谢您！
另外，我想请教一下。
我能理解 minimum=1 代表着最小数量, maximum=32 代表着最大数量, 
但是我不理解 step=1, value=2 这个 value 的含义。
而且我看您举例修改时直接把 value 也修改为 999999 ，但在源码中这个 value = 2  并不是 32 ，所以我这种傻小白就懵逼了。
我仅仅是想请教一下这个 value 在源码中的含义啥意思。
非常抱歉浪费您的时间回复如此笨拙的问题

**Reply by z0brk** [reply]
docs: https://www.gradio.app/docs/slider

**Reply by odinxu** [reply]
非常非常感谢，这个帮了我大忙了。
Thank you very, very much. This will help me a lot.
